templates/terraform/environments/shared/console_allowed.tf

```diff
@@ -0,0 +1,32 @@
+# AWS policy documents for allowing console
+data "aws_iam_policy_document" "console_allowed" {
+  statement {
+    sid       = "AllowConsolePasswordDisplay"
+    effect    = "Allow"
+    actions = [
+      "iam:ListUsers",
+    ]
+    resources = ["arn:aws:iam::${local.account_id}:user/*"]
+  }
+  statement {
+    sid     = "AllowManageOwnProfile"
+    effect  = "Allow"
+    actions = [
+      "iam:GetLoginProfile",
+      "iam:CreateLoginProfile",
+      "iam:DeleteLoginProfile",
+      "iam:UpdateLoginProfile",
+    ]
+    resources = ["arn:aws:iam::${local.account_id}:user/$${aws:username}"]
+  }
+}
+
+resource "aws_iam_group" "console_allowed" {
```

Comment:
This feels like too granular of a group, and something we should just be attaching to users by default but if you want to make it a group that's fine.